src/bsp/src/posix/src/BSP.cpp

```diff
@@ -23,6 +24,9 @@ void rosWatcher(void* param) {
         vTaskDelay(loopRate);
     }
 
+    TCPUartMock& uartMock = static_cast<TCPUartMock&>(BSPContainer::getHostUart());
+    uartMock.close();
```

Comment:
Is it needed since it's called in the destructor? Idk, i'm legit curious for a static object if it's destructor is called

Reply:
yeah i think it's called in the destructor, I was just testing things as ROS had problems shutting down the node.